uber/site_sections/groups.py

```diff
@@ -136,6 +136,26 @@ def index(self, session, message='', order='name', show='all'):
             'approved_tables':   sum(g.tables for g in groups if g.status == c.APPROVED)
         }
 
+    def promo_code_groups(self, session, message=''):
```

Comment:
One day, all our page handlers will basically look like this.

One day.

Reply:
Haha, we could take it a step farther (one day).  Suppose that we have two pages which each look like this:

```python
def promo_code_groups(self, session, message=''):
    groups = session.query(PromoCodeGroup).options(joinedload('buyer')).order_by('name')
        return {
            'groups': groups,
            'message': message,
        }

def mivs_teams(self, session, message=''):
    teams = session.query(MIVSTeam).order_by('name')
    return {
        'groups': groups,
￼	'message': message
    }
```

Most of that is just repeated code.  So we could probably just make a utility method and replace it with something like this:

```python
promo_code_groups = make_page(lambda **params: session.query(PromoCodeGroup).options(joinedload('buyer')).order_by('name'))

mivs_teams = make_page(lambda **params: session.query(MIVSTeam).order_by('name'))
```

Just a thought :)